### PR TITLE
Fix tables generator: rows/columns issue

### PR DESCRIPTION
**Pull request by sunny-viktoryia**
closes: https://github.com/sparkletown/internal-sparkle-issues/issues/811

There were default values for the number of rows and columns set. Although, we need to define the number of rows based on the capacity of the table.

### CODE REVIEW

File: src/utils/table.ts [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import { Table } from "types/Table";
 
+const DEFAULT_COLUMNS_COUNT = 3;
```
Comment (yurii-lubynets) [inline src/utils/table.ts:3]: IMHO it's worth keeping this const in `src/settings.ts`. WDYT?

Comment (0xdevalias) [reply src/utils/table.ts]: Makes sense to do so to me.

**Comment by 0xdevalias** [comment]
**Note:** It probably makes sense to land this other PR first, then solve the minor merge conflicts once it does:

- https://github.com/sparkletown/sparkle/pull/1821

File: src/utils/table.ts (hunk 1 above)
Comment (sunny-viktoryia) [reply src/utils/table.ts]: it's already there -- appeared when I merged changes from staging. `DEFAULT_TABLE_COLUMNS`. 🙃 

(but haven't tested the script after that merge)

**Comment by yurii-lubynets** [comment]
LGTM, looks like @sunny-viktoryia has fixed tables view for `Jazzbar` and `ConversationSpace`

Screenshot 2021-07-16 at 13 18 34

File: src/components/templates/Jazzbar/JazzTab/JazzTab.scss [hunk 2]
```diff
@@ -20,7 +20,7 @@
   .music-bar-content {
     display: grid;
 
-    padding: 10px 70px;
+    padding: 10px 70px 40px;
```
Comment (yurii-lubynets) [inline src/components/templates/Jazzbar/JazzTab/JazzTab.scss:23]: fixed bottom indent for tables on the `Jazzbar` template. They were cropped.

Screenshot 2021-07-16 at 13 26 55

**Comment by yurii-lubynets** [comment]
@sparkletown/engineering feel free to review it

File: src/utils/table.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { DEFAULT_TABLE_ROWS, DEFAULT_TABLE_COLUMNS } from "settings";
```
Comment (mike-lvov) [inline src/utils/table.ts:1]: I dont think we use `DEFAULT_TABLE_ROWS` anymore. If this is the case, delete it from the settings, please

**Review by mike-lvov — APPROVED** [review]

File: src/utils/table.ts (hunk 3 above)
Comment (yurii-lubynets) [reply src/utils/table.ts:1]: `DEFAULT_TABLE_ROWS` is used in calculation of `DEFAULT_TABLE_CAPACITY` in `src/settings.ts`
and in `src/components/molecules/TablesUserList/TablesUserList.tsx`

Comment (mike-lvov) [reply src/utils/table.ts:1]: Okay, sounds good

**Pull request merged**